src/UI/class-settings-page.php

```diff
@@ -543,7 +543,6 @@ function (): void {
 				'title'      => __( 'Track Post Types as', 'wp-parsely' ),
 				'option_key' => $field_id,
 				'help_text'  => $field_help,
-				'filter'     => 'wp_parsely_trackable_statuses',
```

Comment:
This seems wrong. `wp_parsely_trackable_statuses` is to manage post statuses, not types.

Reply:
Indeed!